Fixed FileNotFound: landing_page.json error (#3570)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased
* Fixed an issue where logging arguments were not in the standard kebab-case. The new arguments are: **console-log-threshold**, **file-log-threshold**, **log-file-path**.

> * Fixed an issue where the **upload** command failed because it didn't found the landing_page.json file in content repo.

Reviewer: ```suggestion
* Fixed an issue where the **upload** command failed for private repositories while trying to find the landing_page.json file.
```
